### ISSUE TITLE
Doesn't compile

### ISSUE THREAD

**Issue by 0xd34b33f** [wayfair-tremor/tremor-language-server#22]
Version from cates.io doesnt compile with this error:
```
 Compiling lalrpop v0.18.1
error[E0308]: mismatched types
   --> /home/username/.cargo/registry/src/github.com-1ecc6299db9ec823/simd-json-0.3.3/src/lib.rs:213:86
    |
213 | fn please_compile_with_a_simd_compatible_cpu_setting_read_the_simdjonsrs_readme() -> ! {}
    |    ----------------------------------------------------------------------------      ^ expected `!`, found `()`
    |    |
    |    implicitly returns `()` as its body has no tail or `return` expression
    |
    = note:   expected type `!`
            found unit type `()`
```
In git version there is  lalrpop v0.17.2 and everything works.

**Comment by darach** [comment]
Interesting one Vladimir, we're looking into this now. Thanks for registering the issue!

**Comment by Licenser** [comment]
Heya, we forgot to include the .cargo folder in 0.7.3, I just published 0.7.4 that fixes the issue :)
